.github/workflows/integration-tests.yml

```diff
@@ -134,11 +134,19 @@ jobs:
         uses: shivammathur/setup-php@v2
         with:
           coverage: "none"
-          extensions: "json,couchbase-3.2.2,memcached,mongodb-1.12.0,redis,rdkafka,xsl,ldap"
+          extensions: "json,couchbase-3.2.2,memcached,mongodb-1.12.0,redis,rdkafka,xsl,ldap,msgpack,igbinary"
           ini-values: date.timezone=UTC,memory_limit=-1,default_socket_timeout=10,session.gc_probability=0,apc.enable_cli=1,zend.assertions=1
           php-version: "${{ matrix.php }}"
           tools: pecl
 
+      - name: Install Relay
+        run: |
+          curl -L "https://builds.r2.relay.so/dev/relay-dev-php${{ matrix.php }}-debian-x86-64.tar.gz" | tar xz
+          cd relay-dev-php${{ matrix.php }}-debian-x86-64
+          sudo cp relay.ini $(php-config --ini-dir)
+          sudo cp relay-pkg.so $(php-config --extension-dir)/relay.so
+          sudo sed -i "s/00000000-0000-0000-0000-000000000000/$(cat /proc/sys/kernel/random/uuid)/" $(php-config --extension-dir)/relay.so
```

Comment:
what is this for?

Reply:
It's for telemetry, we use it to identify installations anonymously and what features they use, how long they run. It can be omitted. We are working on getting Relay into `shivammathur/setup-php`.

Reply:
Let me know if I should remove this ;)

Reply:
You can keep it, once the `shivammathur/setup-php` is merged it comes unnecessary.

Reply:
I'd remove it then, this will reduce back and forth on this file

Reply:
Support was merged, but it's not released yet:

https://github.com/shivammathur/setup-php/issues/682#issuecomment-1381168907